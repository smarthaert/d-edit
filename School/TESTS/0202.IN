0
------
1
-------
13
--------
2
--------
16
