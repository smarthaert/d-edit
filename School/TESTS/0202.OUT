0
-------
1
----------
1101
--------
10
--------
10000


